packages/babel-plugin/src/babel-plugin.ts

```diff
@@ -236,7 +237,7 @@ export default declare<State>((api) => {
         const userLandModule = path.node.source.value;
 
         const isCompiledModule = this.importSources.some((compiledModuleOrigin) => {
-          if (userLandModule === DEFAULT_IMPORT_SOURCE || compiledModuleOrigin === userLandModule) {
```

Comment:
Makes sense..

Reply:
Yeah just redundant code. Removed it.

Reply:
i'm confused why `userLandModule === DEFAULT_IMPORT_SOURCE` was checked at all, since `this.importSources` appears to always include `@compiled/react`... am i missing something?

Reply:
ahh just saw your comment @itsdouges yeah looks good